Components/Avatar/avatar.html

```diff
@@ -7,11 +7,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="author" content="Kotesh Mudila">
     <meta name="description" content="Handy-UI documentation - Avatar component">
-    <link rel="stylesheet" href="avatar.css">
-    <link rel="stylesheet" href="/index.css">
+    <link rel="stylesheet" href="/css/index.css">
     <link rel="stylesheet" href="/css/header.css">
     <link rel="stylesheet" href="/css/sidebar.css">
     <link rel="stylesheet" href="/css/components.css">
+    <link rel="stylesheet" href="/css/docs.css">
+
```

Comment:
You can make one combined CSS file and import all these files there. And just use one link tag for importing that combined CSS file